lib/tokens/token.js

```diff
@@ -60,7 +60,6 @@ module.exports = function (log, crypto, P, hkdf, Bundle, error) {
           Token.deriveTokenKeys(TokenType, bytes)
             .then(
               function (keys) {
-                details.createdAt = Date.now()
```

Comment:
Here's the hairy bit. This line was forcing `createdAt` to always be `Date.now()`, rendering the assignment from `details.createdAt` on line 41 redundant. Does it bother anyone that I've removed it?